resources/manifests/pod-checkpointer-cluster-role-binding.yaml

```diff
@@ -10,3 +10,17 @@ subjects:
 - kind: ServiceAccount
   name: pod-checkpointer
   namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
```

Comment:
Same here, although not sure it is worth to create a new file. Maybe use "role" or "roles" for both (ClusterRoleBinding and RoleBinding)?

Reply:
done